study-builder/studies/rarex/study.conf

```diff
@@ -1190,7 +1279,7 @@
       },
       "cancelExpr": """user.studies["rarex"].forms["PARENTAL_CONSENT"].hasInstance()
         && user.studies["rarex"].forms["PARENTAL_CONSENT"].isStatus("COMPLETE")""",
-      "delaySeconds": 420,
+      "delaySeconds": 604800,
```

Comment:
`420 sec / 60` is 7 minutes. What we want is 7 days (aka 604800 secs). This is likely a leftover from local testing.